textnow_bot.py

```diff
@@ -44,3 +47,56 @@ def send_message(self, recipient, message):
         self.page.type("#text-input", message)
         self.page.press("#text-input", "Enter")
         self.page.waitForTimeout(500)
+
+
+# Code for Async Below
+
+class AsyncMeta(type):
```

Comment:
```suggestion
class AsyncMeta(type): # Allows the __init__ function of AsyncTextNowBot() to be async.
```

`AsyncMeta` is a class I created to get around the fact that normally, the `__init__` function of a class cannot be asynchronous in Python. 

This issue could also have been solved by creating a `log_in` function, and transferring all the code out of the `__init__` function into the new `log_in` function, but this would mean that instead of using the following code to send a message:

```python
bot = await AsyncTextNowBot(page, None, username, password)
await bot.send_message(recipient, message)
```

You would have to do:

```python
bot = await AsyncTextNowBot(page, None, username, password)
await bot.log_in()
await bot.send_message(recipient, message)
```

Alternatively, you could have the `__init__` function include `asyncio.run(log_in())`, which would avoid the extra command for the user. 

However, I thought that for consistency it would make more sense for each of the two classes (`TextNowBot` and `AsyncTextNowBot`) to have the same methods, so I decided on using the metaclass.


Thanks for addressing the linter concerns.

Reply:
I see, that makes a lot of sense. To give you some context, I specifically added login logic to `__init__` so that functions like `send_message` have the invariant that the user has logged in. It simplifies the need to keep track of states (whether the user is logged in or not, etc.)

I'm not a fan of `AsyncMeta`, because it looks hack-y to me. I feel like I would prefer to just change the API in this case to add a `log_in` method so that both sync_api and async_api have the same experience, and we can do `async log_in()` like you suggested. Overall, it's a cleaner solution in my opinion, but I don't have any strong opinions about that. What do you think?

Reply:
> I feel like I would prefer to just change the API in this case to add a `log_in` method ...

Yeah, that sounds fine to me. I'll put that together.